data/categories.txt

```diff
@@ -757,7 +757,7 @@ pl:^Sklep spożywczy
 pt:^Loja de conveniência
 pt-BR:^Loja de conveniência|4mercadinho|compras
 ro:4^Magazin mixt
-es:^Tienda de abarrotes|chino|bazar
+es:^Tienda|chino|bazar
```

Comment:
Are you sure that "chino" and "bazar" are relevant here?
http://wiki.openstreetmap.org/wiki/Tag:shop%3Dconvenience